frontend/src/app/shared/box/box.component.html

```diff
@@ -4,7 +4,10 @@
       <ng-container *ngIf="titleTemplateRef; else stringTitle">
         <ng-container *ngTemplateOutlet="titleTemplateRef"></ng-container>
       </ng-container>
-      <ng-template #stringTitle>{{ boxTitle | translate }}</ng-template>
+      <ng-template #stringTitle>
+        <img class="oib-title-image" *ngIf="imagePath" [src]="imagePath" alt="Image" /> <!-- Affiche l'image correspondante si imagePath n'est pas vide -->
```

Comment:
Don't put french comments: an open source project (and an IT project in general) may include non french speaking people, so this comment will be more difficult to handle for them